core/modules/installer/installer.theme.inc

```diff
@@ -127,6 +127,15 @@ function template_preprocess_installer_browser_project_dialog(&$variables) {
   $variables['project_page'] = l(t('View on BackdropCMS.org'), 'https://backdropcms.org/project/' . $project['name']);
   $variables['description'] = _filter_htmlcorrector(filter_xss($project['description'], array('h2', 'p', 'a', 'ul', 'li', 'pre')));
   $variables['title'] = $project['title'];
+  $variables['image'] = FALSE;
+
+  // If the project has an image, render it.
+  if (!empty($project['image'])) {
+    $variables['image'] = theme('image', array(
+      'path' => $project['image'],
+      'alt' => t($project['title']),
```

Comment:
More so:
>You should never use t() to translate variables, such as calling t($text);

From Backdrop API docs for `t()`

Reply:
Right you are. Especially in a situation like this where the project title is almost certainly not going to be in the site's existing translation list (nor should it be). And project titles typically aren't translated anyway, as we treat them as proper nouns in most cases.

Since I now "own" this PR, I've made these fixes and repushed.